pkg/volume/nfs/nfs.go

```diff
@@ -245,7 +245,7 @@ func (nfsMounter *nfsMounter) SetUp(mounterArgs volume.MounterArgs) error {
 
 func (nfsMounter *nfsMounter) SetUpAt(dir string, mounterArgs volume.MounterArgs) error {
 	notMnt, err := mount.IsNotMountPoint(nfsMounter.mounter, dir)
-	klog.V(4).Infof("NFS mount set up: %s %v %v", dir, !notMnt, err)
+	klog.V(4).InfoS("NFS mount set up", "path", dir, "!notMnt", !notMnt, "err", err)
```

Comment:
We cannot just dump variable names as we cannot assume that people that read logs need to know code.
```suggestion
	klog.V(4).InfoS("NFS mount set up", "path", dir, "isMountPoint", !notMnt, "err", err)
```

Reply:
Yeah,  `isMountPoint` is more readable for common users. This has been modified according to the suggestion.